code/controllers/subsystems.dm

```diff
@@ -0,0 +1,46 @@
+#define NEW_SS_GLOBAL(varname) if(varname != src){if(istype(varname)){Recover();qdel(varname);}varname = src;}
+
+/datum/subsystem
```

Comment:
The alarm handler was made before someone ported us the process scheduler. /datum/subystem shouldn't be in use at this point.